hw/arc/arc_virt.c

```diff
@@ -27,6 +27,7 @@
  * their name, parent structure, name inside parent structure, and default value
  */
 static Property arc_virt_properties[] = {
+    DEFINE_PROP_UINT64("ram-start", ARCVirtMachineState, ram_start, VIRT_RAM_BASE),
```

Comment:
Since we use that value in that one place only I'd suggest to use explicit value just here, so it's immediately obvious what that value is, rather than having a definition in another header file. I.e. just use:
```c
DEFINE_PROP_UINT64("ram-start", ARCVirtMachineState, ram_start, 0x80000000)
```
And add in-place comment like that: for compatibility reason's assume default memory start at 0x8000_0000 (2 GiB), if needed change with command line option "-M virt,ram-start=0x12345678".